algo-dev-demo/digits-recognition/README.md

```diff
@@ -0,0 +1,104 @@
+# Digit recognition demo
+
+In this demo we'll go through deploying a pre-trained [scikit-learn](https://scikit-learn.org/) model for hand written digits recognition and we will explore some options to query this model to make inferences.
+
+The trained model is based on the [scikit-learn documentation](https://scikit-learn.org/stable/auto_examples/classification/plot_digits_classification.html).
+
+This demo features:
+
+- Hosting models in the Algorithmia platform
+- Calling other algorithms as part of the inference pipeline, in this case for image preprocessing
+- Use of multiple query types, in this case: url and base64
+
+## Prerequisites
+
+Fork this repository so you have access to the algorithm code and data files. We will use the Algorithmia Web IDE in this demo but note that you can also use a repository [from Github](https://algorithmia.com/developers/algorithm-development/source-code-management) or you can use the [CLI](https://algorithmia.com/developers/clients/cli/) to deploy your model instead once you've [created an algorithm](https://algorithmia.com/developers/algorithm-development/languages/python/#create-an-algorithm) and cloned your repository.
+
+## Upload pre-trained model to Data Collections
+
+We are going to host the serialized pre-trained model on the Algorithmia platform, for more info look at the [Data Collections docs](https://algorithmia.com/developers/data/hosted/).
```

Comment:
nit: Can drop the trailing slash here to avoid a redirect.
```suggestion
We are going to host the serialized pre-trained model on the Algorithmia platform, for more info look at the [Data Collections docs](https://algorithmia.com/developers/data/hosted).
```